From ad9ac7fd8a3495985dbd745ca71c8aa4b791aaed Mon Sep 17 00:00:00 2001
From: kopal <kopal@broadcom.com>
Date: Tue, 18 Jun 2013 17:16:53 +0530
Subject: dts: Add "cpld" parameter.

The "cpld" parameter is added and set to "yes". If the field is set to "no"
or no such parameter is mentioned in the dts, nae will be initialized based
on the compex settings in dts.
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/arch/mips/boot/dts/xlp2xx-tiny.dts b/arch/mips/boot/dts/xlp2xx-tiny.dts
index af9a8f0..c0b87ec 100644
--- a/arch/mips/boot/dts/xlp2xx-tiny.dts
+++ b/arch/mips/boot/dts/xlp2xx-tiny.dts
@@ -95,6 +95,7 @@
 		complex@0 {
 			device_type = "nae-complex";
 			mode = "sgmii";
+			cpld = "yes";
 			/*
 			xaui {
 				loopback = <0>;
diff --git a/arch/mips/boot/dts/xlp3xx-tiny.dts b/arch/mips/boot/dts/xlp3xx-tiny.dts
index 36dd871..63a1c94 100644
--- a/arch/mips/boot/dts/xlp3xx-tiny.dts
+++ b/arch/mips/boot/dts/xlp3xx-tiny.dts
@@ -95,6 +95,7 @@
 		complex@0 {
 			device_type = "nae-complex";
 			mode = "sgmii";
+			cpld = "yes";
 			/*
 			xaui {
 				loopback = <0>;
diff --git a/arch/mips/boot/dts/xlp8xx-tiny.dts b/arch/mips/boot/dts/xlp8xx-tiny.dts
index 1688147..e7db6ea 100644
--- a/arch/mips/boot/dts/xlp8xx-tiny.dts
+++ b/arch/mips/boot/dts/xlp8xx-tiny.dts
@@ -97,6 +97,7 @@
 		complex@0 {
 			device_type = "nae-complex";
 			mode = "sgmii";
+			cpld = "yes";
 			xaui {
 				loopback = <0>;
 
diff --git a/arch/mips/boot/dts/xlp9xx-tiny.dts b/arch/mips/boot/dts/xlp9xx-tiny.dts
index 5b14c63..d08c202 100644
--- a/arch/mips/boot/dts/xlp9xx-tiny.dts
+++ b/arch/mips/boot/dts/xlp9xx-tiny.dts
@@ -71,6 +71,7 @@
 				};
 				complex@0 {
 					mode = "xgmac"; /* sgmii only */
+					cpld = "yes";
 				      xgmac {                          /* valid when mode = xaui, rxaui or xfi */
 
 						loopback = <0>;             /* loopback = 0 or 1 */
-- 
1.7.1

